samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -179,27 +181,27 @@ public synchronized void stop() {
     boolean containerShutdownInvoked = false;
     if (container != null) {
       try {
-        LOGGER.info("Shutting down container " + container.toString() + " from StreamProcessor");
+        LOGGER.info("Shutting down the container: {} of stream processor: {}.", container, processorId);
         container.shutdown();
+        LOGGER.info("Waiting {} milliseconds for the container: {} to shutdown.", taskShutdownMs, container);
         containerShutdownInvoked = true;
-      } catch (IllegalContainerStateException icse) {
-        LOGGER.info("Container was not running", icse);
+      } catch (Exception exception) {
+        LOGGER.error(String.format("Shutdown of container: %s threw an exception.", container), exception);
```

Comment:
Mention that you're ignoring this.

"Ignoring error during shutdown of container."